values.yaml

```diff
@@ -434,6 +434,25 @@ mastodon:
         enabled: false
         port: 9102
 
+    # Settings for Prometheus metrics.
+    # For more information, see:
+    # https://docs.joinmastodon.org/admin/config/#prometheus
```

Comment:
Maybe add that this requires running Mastodon 4.4?

Reply:
You mean 4.3? Or will this only be available in 4.4?

Reply:
4.4 only, it is not in 4.3